tests/test_threading.py

```diff
@@ -168,6 +175,7 @@ def test_model_context(self):
 
 
 # Same as TestLockedTransition but with LockedHierarchicalMachine
+@skipIf(pgv is None, 'AGraph diagram requires pygraphviz')
```

Comment:
yeah, I first wondered what 'S' could mean :). The only thing (I assume) you have to do to make the tests in `test_threading` work without pygraphviz is to change this line 
```python
self.stuff = Stuff(states, machine_cls=MachineFactory.get_predefined(locked=True, graph=True, nested=True))
```
into 

```python
self.stuff = Stuff(states, machine_cls=MachineFactory.get_predefined(locked=True, nested=True))
```

in  `setUp` of `TestLockedHierarchicalTransitions`.